### ISSUE TITLE
提供的模型参数与代码中模型参数不匹配

### ISSUE THREAD

**Issue by zmghub** [wuzhenyubuaa/SODGAN#1]
您好，感谢您的工作。我在试图跳过step1 执行step2的时候，发现您提供的网盘中下载的biggan模型参数文件仅有3.6M，并且其中保存的参数仅有layers.0、layers.2、layers.3、layers.5、layers.6这几个层，与模型定义文件中所需的参数相差巨大，麻烦您帮忙检查一下提供的链接？

**Comment by zmghub** [comment]
你好，您的邮件我已收到，谢谢

**Comment by wuzhenyubuaa** [comment]
@zmghub ，感谢您的关注。我提供的那个3.6MB的文件是Saliency mask generator的pretrained model，BigGAN的预训练模型可前往官方实现去下载，我给你提供个地址 https://github.com/huggingface/pytorch-pretrained-BigGAN/blob/master/pytorch_pretrained_biggan/model.py，   里边包含了链接。

**Comment by wuzhenyubuaa** [comment]
同样的StyleGAN的预训练模型也可以在其官方实现下载的
